activerecord/test/cases/associations/has_many_through_associations_test.rb

```diff
@@ -889,7 +889,8 @@ def test_collection_singular_ids_setter_raises_exception_when_invalid_ids_set
     company = companies(:rails_core)
     ids = [Developer.first.id, -9999]
     e = assert_raises(ActiveRecord::RecordNotFound) { company.developer_ids = ids }
-    assert_match(/Couldn't find all Developers with 'id'/, e.message)
+    msg = "Couldn't find all Developers with 'id': (1, -9999) (found 1 results, but was looking for 2).Please check following 'id': -9999."
```

Comment:
So the space should go at the beginning of the next sentence. Otherwise if there aren't found ID's you end up with:

"This is a sentence. " instead of "This is a sentence."